tests/array.rs

```diff
@@ -1970,6 +1970,77 @@ fn array_macros() {
     assert_eq!(empty2, array![[]]);
 }
 
+#[cfg(test)]
+mod as_standard_layout_tests {
+    use super::*;
+
+    #[test]
+    fn test_f_layout() {
+        let shape = Ix2(2, 2).strides(Ix2(1, 2));
```

Comment:
Fwiw, this can be written `(2, 2).strides((1, 2))` or, since it's Fortran layout in this case, `(2, 2).f()`.